main.nf

```diff
@@ -1148,6 +1177,61 @@ process split_normalize {
 
 }
 
+/////////////// Collect QC into single file ///////////////
+
+process merge_qc_json {   
+  	cpus 1
+	errorStrategy 'retry'
+	maxErrors 5
+    publishDir "${OUTDIR}/qc", mode: 'copy' , overwrite: 'true', pattern: '*.QC'
+	tag "$id"
+	time '10m'
+    stageInMode 'copy'
+	stageOutMode 'copy'
```

Comment:
again some weird indendations

Reply:
would also remove stageIn/out

This is a small process and since this is run on the ssd-cluster it makes no difference. The read and write actually becomes worse with stage-in and stage-out for small files

Reply:
Fixed on both counts!